lamindb/_feature.py

```diff
@@ -167,11 +167,14 @@ def process_init_feature_param(args, kwargs, is_param: bool = False):
     dtype: type | str | None = kwargs.pop("dtype") if "dtype" in kwargs else None
     is_type: bool = kwargs.pop("is_type") if "is_type" in kwargs else False
     type_: Feature | str | None = kwargs.pop("type") if "type" in kwargs else None
+    description = kwargs.pop("description") if "description" in kwargs else None
     if kwargs:
         valid_keywords = ", ".join([val[0] for val in _get_record_kwargs(Feature)])
         raise FieldValidationError(f"Only {valid_keywords} are valid keyword arguments")
     kwargs["name"] = name
     kwargs["type"] = type_
+    if not is_param:
```

Comment:
Why do we need this here?

Features and Params should behave in the same way when it comes to descriptions?